### PR TITLE
[202012][orchagent]: Handle duplicate routes in a graceful manner

### PR DESCRIPTION
**Pull request by prabhataravind**
* With VNET routes, the same route can be learned via multiple sources, like via BGP. Handle this special situation gracefully without orchagent crash

**What I did**
Handle duplicate routes learned via multiple sources
**Why I did it**
To prevent orchagent crashes and thereby improve system stability
**How I verified it**

**Details if related**

### CODE REVIEW

File: orchagent/orch.cpp [hunk 1]
```diff
@@ -853,6 +853,22 @@ task_process_status Orch::handleSaiCreateStatus(sai_api_t api, sai_status_t stat
                     exit(EXIT_FAILURE);
             }
             break;
+        case SAI_API_ROUTE:
+            switch (status)
+            {
+                case SAI_STATUS_SUCCESS:
+                    SWSS_LOG_WARN("SAI_STATUS_SUCCESS is not expected in handleSaiCreateStatus");
+                    return task_success;
+                case SAI_STATUS_ITEM_ALREADY_EXISTS:
```
Comment (prsunny) [inline orchagent/orch.cpp:862]: please also handle SAI_STATUS_ITEM_NOT_FOUND in handleSaiRemoveStatus

**Comment by prsunny** [comment]
Please add a test to add two routes and delete route one-by-one

**Review by prsunny — APPROVED** [review]

File: orchagent/orch.cpp (hunk 1 above)
Comment (prabhataravind) [reply orchagent/orch.cpp:862]: Done

File: tests/conftest.py [hunk 2]
```diff
@@ -389,10 +389,15 @@ def __init__(
         if buffer_model == 'dynamic':
             enable_dynamic_buffer(self.get_config_db(), self.runcmd)
 
-    def destroy(self) -> None:
-        if self.appldb:
+    def del_appl_db(self):
```
Comment (prsunny) [inline tests/conftest.py:392]: Please provide the original PR that you cherry-picked  this.

**Comment by prsunny** [comment]
@prabhataravind , can you please rebase to latest?

**Comment by prabhataravind** [comment]
> @prabhataravind , can you please rebase to latest?

Done

**Review by theasianpianist — APPROVED** [review]

**Pull request merged**